oshc/main/templates/contest_edit.html

```diff
@@ -7,7 +7,7 @@
         <div class="form-group">
             <label style="margin-top:70px;" for="name" class="control-label col-xs-2">Name:</label>
             <div class="col-xs-3">
-                <input type="text" style="margin-top:70px;" class="form-control" id="name" name="name" placeholder="Contest Name" required>
+                <input type="text" style="margin-top:70px;" class="form-control" id="name" pattern="([A-Z, ])\w+" name="name" placeholder="Contest Name" required>
```

Comment:
[A-Z, ] won't go well if the user submits contest name which only has small characters. ```[A-Za-z.\s-]``` would be better. Can you please update this one?